### DISCUSSION TITLE
课程三中，如果我的账户没有1000USD，我只有900，最后取不出来了，那我钱就一直在合约中吗？

### DISCUSSION THREAD

**Discussion by henry652341974** [smartcontractkit/Web3_tutorial_Chinese#11] [Q&A] [answered]
课程三中，如果我的账户没有1000USD，我只有900，最后取不出来了，那我钱就一直在合约中吗？

**Comment by henry652341974** [comment]
我把合约叉掉了，钱没了。。。

**Comment by henry652341974** [comment]
0x64fB697e5e8759B10391c0080F00C070Dd557dA8

给我点钱把，部署不起来了，一直pending

为啥每次只能领0.1ETH，太少了哦

**Comment by henry652341974** [comment]
![image](https://github.com/user-attachments/assets/971d8bd1-b30b-4704-af42-6eecae264ca3)

应该是这个he'yue合约地址 ，这个钱是不是死里面出不来了？

**Comment by QingyangKong** [comment, marked as answer]
给你转了，后面缺了，可以在B站留地址，会定期打。
如果是没有写 withdraw 函数，就取不出来了。

**Comment by henry652341974** [comment]
感谢感谢，我现在还是有点纠结和迷茫，现在国内法律上确实对于参与DAPP和区块链相关的远程工作是不合法的，到底要不要继续学并转型，还是很困惑，求解
&nbsp;


/ds…Ⅶ乄、
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Frank ***@***.***&gt;; 
发送时间: 2024年11月25日(星期一) 下午2:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [smartcontractkit/Web3_tutorial_Chinese] 课程三中，如果我的账户没有1000USD，我只有900，最后取不出来了，那我钱就一直在合约中吗？ (Discussion #11)














 
给你转了，后面缺了，可以在B站留地址，会定期打。
 如果是没有写 withdraw 函数，就取不出来了。

**Reply by QingyangKong** [reply]
GitHub 不提供非技术问题的建议，这个要自己决定！
